viewer-config-persistence/src/main/java/nl/b3p/viewer/config/services/GeoService.java

```diff
@@ -74,7 +74,9 @@
 
     private String username;
     private String password;
-
+    
+    private String geofenceHeader;
```

Comment:
I think this adds an extra columnt in the datamodel, no?
In that case upgrade scripts must be provided to do that, see https://github.com/flamingo-geocms/flamingo/tree/master/viewer-config-persistence/src/main/resources/scripts

Reply:
Yes you are right.